edweiss-web/src/app/(app)/(board)/course/[id]/index.tsx

```diff
@@ -0,0 +1,66 @@
+import ReactComponent, { ApplicationRoute } from '@/constants/Component';
+import { IconFileTypePdf } from '@tabler/icons-react';
+import { ReactNode } from 'react';
+
+const IndexRoute: ApplicationRoute = () => {
```

Comment:
This is very nit-picky, but you can use variables to store the text, so that the ui is more readable.

Reply:
Hello,
This is just a placeholder, of course all the text will be replaced by fetched data :) I'll get to it next sprint!